src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -631,189 +610,83 @@ public TypeSymbolWithAnnotations SetUnknownNullabilityForReferenceTypes()
 
         private static TypeSymbolWithAnnotations CreateWithUnknownNullability(TypeSymbol typeSymbol, ImmutableArray<CustomModifier> customModifiers)
```

Comment:
Consider removing this method. Callers can do `Create(typeSymbol, isNullable: null, customModifiers)`.